ena_upload/ena_upload.py

```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env python
 __authors__ = ["Dilmurat Yusuf", "Bert Droesbeke"]
 __copyright__ = "Copyright 2020, Dilmurat Yusuf"
+__maintainer__ = "Bert Droesbeke"
 __email__ = "Dilmurat.yusuf@gmail.com"
```

Comment:
change to your email?

Reply:
I don't know, the biggest part of the base code is still thanks to Dilmurat, so I also don't want to steal his thunder...

Reply:
Yes, but people should probably contact you, left?

Reply:
that is also true :) Will change it